File: Nivel_3/Retos/ParImpar.java
public class ParImpar{
    public static void main(String[] args) {
        int valor = 5;
        do{
            if(valor % 2 == 0){System.out.println("El valor " + valor + " es par.");}
            else {System.out.println("El valor " + valor + " es impar.");}
        }while(valor < 0);
        System.out.println(valor + " es negativo.");
    }
}

File: Nivel_3/Retos/ParImpar.py
n = int(input("Introduce un valor: "))
if n < 0:
	print("El valor es negativo. Finalizando programa.")
elif n % 2 == 0:
	print("El valor " + str(n) + " es par.")
else:
	print("El valor " + str(n) + " es impar.")
